runtime/src/bank.rs

```diff
@@ -1638,7 +1638,7 @@ impl Bank {
         }
 
         self.collected_rent
-            .fetch_add(collected_rent, Ordering::Relaxed);
+            .fetch_add(dbg!(collected_rent), Ordering::Relaxed);
```

Comment:
dbg?

Reply:
![image](https://user-images.githubusercontent.com/1281082/80762294-1ad2de80-8af1-11ea-8819-32489c1dab4a.png)
I caught that in the next commit